### PR TITLE
clowdapp: up memory resource and lower workers

### PR DESCRIPTION
**Pull request by loadtheaccumulator**
Signed-off-by: Jonathan Holloway <jholloway@redhat.com>

# Description

Changing memory and worker resources

Fixes # (issue)

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `go fmt ./...` to check that my code is properly formatted
- [ ] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

**Review by omaciel — APPROVED** [review]
lgtm

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
02153c4a-4f8b-4f47-93e9-87d3e4a6ee29
323e6a5e-e502-452d-89de-5a5366c70bd9

File: deploy/clowdapp.yaml [hunk 1]
```diff
@@ -217,7 +217,7 @@ parameters:
 - description: Number of workers for uploading to a backing object storage bucket
   name: UPLOADWORKERS
   required: false
-  value: "100"
+  value: "50"
```
Comment (AaronH88) [inline deploy/clowdapp.yaml:220]: This should be one of the first things to attempt to increase

**Review by AaronH88 — APPROVED** [review]
lgtm

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
3de25a11-a093-49be-a09a-f17bf4d198a7
440fe048-9df9-4298-8dca-a7452a0577c7

**Pull request merged**